classes/controller/AdminController.php

```diff
@@ -313,9 +313,14 @@ public function __construct()
 		if ($this->multishop_context == -1)
 			$this->multishop_context = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
 
-		$this->bo_theme = ((Validate::isLoadedObject($this->context->employee) && $this->context->employee->bo_theme) ? $this->context->employee->bo_theme : 'default');
+        // Fix #PSCSX-4339 - "default" theme name should be customizable
```

Comment:
Hi,

thank you for your PRs.
Can you please use tabs as indentation ?

Regards